components/scream/src/share/tests/vertical_interp_tests.cpp

```diff
@@ -110,16 +256,16 @@ TEST_CASE("main_vertical_interpolation_test"){
   });
   Kokkos::fence();
   
-  for(int col=0; col<3; col++){
-    for(int lev=0; lev<4; lev++){
+  for(int col=0; col<2; col++){
+    for(int lev=0; lev<17; lev++){
       REQUIRE(out_1d_test_s(col,lev) == correct_val[col][lev]);
     }
   }
 
   //Check to see if choose different masked value than default that it returns as expected
-  auto out_same = view_2d<Pack<Real,1>>("",3,npacks_tgt);
+  auto out_same = view_2d<Pack<Real,N>>("",2,npacks_tgt);
```

Comment:
I'm not sure what this test is doing. Are you testing that a mask value of -999 would yield the same result as the default mask value? If that's the case, aren't you basically checking that no value gets masked? And if so, you could just do that by checking the mask view above.

Reply:
This test is testing that a user can specify a mask value different than the default one in scream_vertical_interpolation.hpp. In this case the user provides -999. and this test checks that the function properly uses this and masks out-of-bounds values to -999 (not the default value in the scream_vertical_interpolation.hpp file). I don't think you can check this by checking the mask view above. The mask view above will be the same but your output view will have different values depending on what the user specifies (or doesn't) for the mask value.